DutchFilter/sections/specific.txt

```diff
@@ -215,6 +215,7 @@ klusidee.nl##td.td-ad
 ||pcnltelecom.tdsapi.com/portal/iframe.js
 ||serving-sys.com/*.mp4^$domain=kijk.nl
 ||totaaltv.nl/upload/Banner-sep-astrasat.gif
+beta.9292.nl##div[data-testid="gpt-ad-root"]:upward(1)
```

Comment:
Looks good to me, but I would prefer to use `:has()` instead of `:upward()`, because the former is natively supported by browsers.
Could it be changed to something like this:
```suggestion
beta.9292.nl##:is(div, aside, astro-island):has(> div[data-testid="gpt-ad-root"])
```